diff --git a/sandbox/snek/snek.c b/sandbox/snek/snek.c
index 3e6ed8b..6623198 100644
--- a/sandbox/snek/snek.c
+++ b/sandbox/snek/snek.c
@@ -133,7 +133,7 @@ static void GridFlush(void)
         for (int col = 0; col < GRID_COLS; ++col) {
             if (front_valid && back[row][col] == front[row][col])
                 continue;
-            fprintf(stdout, "\033[%d;%dH%c", row, col, back[row][col]);
+            fprintf(stdout, "\033[%d;%dH%c", row + 1, col + 1, back[row][col]);
             front[row][col] = back[row][col];
         }
     }
