diff --git a/sandbox/wordcount/wordcount.c b/sandbox/wordcount/wordcount.c
index e7d8151..ba2ec6d 100644
--- a/sandbox/wordcount/wordcount.c
+++ b/sandbox/wordcount/wordcount.c
@@ -96,12 +96,15 @@ static void CountStream(FILE *f, struct Counts *out)
 {
     struct Buffer line;
     BufferInit(&line);
+    char last = '?';
     int in_word = 0;
     int c;
 
     while ((c = fgetc(f)) != EOF) {
         ++out->bytes;
         if (c == '\n') {
+            if (line.len > 0)
+                last = *BufferAt(&line, line.len - 1);
             BufferPush(&line, '\0');
             unsigned long line_len = LineLen(line.data);
             if (line_len > out->longest)
@@ -121,12 +124,14 @@ static void CountStream(FILE *f, struct Counts *out)
     }
 
     if (line.len > 0) {
+        last = *BufferAt(&line, line.len - 1);
         BufferPush(&line, '\0');
         unsigned long line_len = LineLen(line.data);
         if (line_len > out->longest)
             out->longest = line_len;
     }
 
+    fprintf(stderr, "last char: '%c'\n", last);
     BufferFree(&line);
 }
 
