diff --git a/sandbox/ds-kit/ds-kit.c b/sandbox/ds-kit/ds-kit.c
index ff3ce5c..9837579 100644
--- a/sandbox/ds-kit/ds-kit.c
+++ b/sandbox/ds-kit/ds-kit.c
@@ -55,7 +55,7 @@ static int (*g_cmp)(const struct Item *, const struct Item *);
 
 static int CmpBridge(const void *pa, const void *pb)
 {
-    return g_cmp((const struct Item *)pa, (const struct Item *)pb);
+    return g_cmp((const struct Item *)pb, (const struct Item *)pa);
 }
 
 void DaSort(struct DynArray *da, int (*cmp)(const struct Item *, const struct Item *))
